### PR TITLE
Cope with trailing whitespace in step markdown

### PR DESCRIPTION
**Pull request by sihugh**
At present, steps with multiple trailing lines will create an empty paragraph once parsed.

This breaks Google's [rich snippet parser](https://search.google.com/test/rich-results?id=sm1WcP0wSd_gSZ-3ZSSVUQ) for HowTo schemas, so we should fix it.

Screenshot 2019-05-30 15 52 56

Note that it doesn't have any visible effects on the page as we seem to defend against that in the step by step component.

https://collections-publisher.publishing.service.gov.uk/step-by-step-pages/66/internal-change-notes

### CODE REVIEW

File: spec/services/step_content_parser_spec.rb [hunk 1]
```diff
@@ -47,6 +47,7 @@
 
           Conducted by Mr Andrew Preview
 
+
```
Comment (leenagupte) [inline spec/services/step_content_parser_spec.rb]: Nitpick police: Why was this line added?

Comment (sihugh) [reply spec/services/step_content_parser_spec.rb]: That's because the single blank line didn't reveal the issue :-(
